src/utils/types.ts

```diff
@@ -332,16 +334,21 @@ interface PageNEW {
   name: string
   progress: Progress
   reviewProgress?: ReviewProgress
+  changeRequestsProgress?: ChangeRequestsProgress
   state: PageElement[]
 }
 
 type PageElement = {
   element: ElementStateNEW
   response: ResponseFull | null
+  previousApplicationResponse: ApplicationResponse
+  latestApplicationResponse: ApplicationResponse
```

Comment:
Overall I think it's better not to use the GraphQL type here, one reason is that it carry all the connections and it is so big, we don't need all that. See if you agree and we can extract just what is required to add here.

I'm still trying to get if we still use the `response` field for displaying or not - for example when the Applicant is filling the form for the first time. I guess it would be using it? But perhaps we can move to only use the latest? Not to do as part of this PR anyway, just trying to see the changes.

Reply:
I am keen to have a conversation about generated types and GraphQL types, I think we can discuss it here: #456